<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exp1</class>
 <widget class="QWidget" name="exp1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="podtv">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:8px;
font: 8pt &quot;Century Gothic&quot;;
}
QPushButton:hover:!pressed
{
background-color: rgb(130,240,255);
}

</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="real">
   <property name="geometry">
    <rect>
     <x>-160</x>
     <y>190</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QLineEdit
{
background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color: rgb(130,240,255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="first">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>199</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
font: 8pt &quot;Century Gothic&quot;;

</string>
   </property>
   <item>
    <property name="text">
     <string>Форма первого числа</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Алгебраическая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Тригонометрическая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Экспоненциальная</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="real_push">
   <property name="geometry">
    <rect>
     <x>-160</x>
     <y>190</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(237, 189, 132,0);
border: 1px solid rgb(40, 91, 174);
border-radius:8px;
font: 75 12pt &quot;System&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="per">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:8px;
font: 8pt &quot;Century Gothic&quot;;
}
QPushButton:hover:!pressed
{
background-color: rgb(130,240,255);
}

</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imag">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QLineEdit
{
background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color: rgb(130,240,255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="real_push_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(237, 189, 132,0);
border: 1px solid rgb(40, 91, 174);
border-radius:8px;
font: 75 12pt &quot;System&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rez">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="oper">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>450</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
font: 8pt &quot;Century Gothic&quot;;

</string>
   </property>
   <item>
    <property name="text">
     <string>Операции</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>√</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Возведение в целую степень(pow)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Log(10)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Log(e)</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="pow">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QLineEdit
{
background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:8px;
font: 75 12pt &quot;System&quot;;
}
QLineEdit:hover:!pressed
{
background-color: rgb(130,240,255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="trigout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="alg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>420</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 20pt &quot;MS Serif&quot;;

</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Serif'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#69ffb4;&quot;&gt;Введите число и выберите действие &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="alg_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>420</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 20pt &quot;MS Serif&quot;;

</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Serif'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#69ffb4;&quot;&gt;Выберите форму числа &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AAA1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>377</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:8px;
font: 8pt &quot;Century Gothic&quot;;
}
QPushButton:hover:!pressed
{
background-color: rgb(130,240,255);
}

</string>
   </property>
   <property name="text">
    <string>Очистить </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ramka">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>500</width>
     <height>140</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119,221,255);
border: 1px solid rgb(255,94,55);
border-radius:20px;
font: 8pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="plainText">
    <string>                   Ответ в Алгебраической форме


                 Ответ в Тригонометрической форме


                  Ответ в Экспоненциальноой форме</string>
   </property>
  </widget>
  <zorder>fon</zorder>
  <zorder>alg_2</zorder>
  <zorder>alg</zorder>
  <zorder>ramka</zorder>
  <zorder>per</zorder>
  <zorder>podtv</zorder>
  <zorder>real</zorder>
  <zorder>first</zorder>
  <zorder>real_push</zorder>
  <zorder>imag</zorder>
  <zorder>real_push_2</zorder>
  <zorder>oper</zorder>
  <zorder>pow</zorder>
  <zorder>trigout</zorder>
  <zorder>AAA1</zorder>
  <zorder>clear</zorder>
  <zorder>rez</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>imag</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>pow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>real</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>-112</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
